src/VisualStudio/Core/Def/EditorConfigSettings/Formatting/View/FormattingBoolSettingView.xaml.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.CodeAnalysis.Editor.EditorConfigSettings.Data;
+
+namespace Microsoft.VisualStudio.LanguageServices.EditorConfigSettings.Formatting.View
+{
+    /// <summary>
+    /// Interaction logic for WhitespaceValueSettingControl.xaml
+    /// </summary>
+    internal partial class FormattingBoolSettingView : UserControl
+    {
+        private readonly FormattingSetting _setting;
+        private readonly CheckBox _checkBox;
+
+        public FormattingBoolSettingView(FormattingSetting setting)
+        {
+            InitializeComponent();
+            _setting = setting;
+            _checkBox = new CheckBox();
```

Comment:
nit: Why is this not just already in the xaml file as a named checkbox? 

Could ignore xaml file altogether if you want to do it this way.